howto/workers.md

```
@@ -0,0 +1,292 @@
+{"level":"error","msg":"worker panicked","worker":"my-worker","event":"..."}
+{"level":"warning","msg":"worker terminated","worker":"my-worker","event":"..."}
+{"level":"warning","msg":"worker backoff","event":"..."}
+{"level":"info","msg":"worker resumed","event":"..."}
+```
+
+## ColdBrew Integration (Phase 2)
+
+The workers package is standalone — any Go service can use it. ColdBrew integration via `CBServiceV2` is planned for a future core release, where workers will be started/stopped as part of the ColdBrew service lifecycle.
+
+[workers]: https://github.com/go-coldbrew/workers
+[workers-docs]: https://pkg.go.dev/github.com/go-coldbrew/workers
```

markdownlint-cli2 (0.22.0)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "workers-docs"

(MD053, link-image-reference-definitions)